share/completions/git.fish

```diff
@@ -645,5 +645,32 @@ complete -f -c git -n '__fish_git_using_command clean' -s x -d 'Remove ignored f
 complete -f -c git -n '__fish_git_using_command clean' -s X -d 'Remove only ignored files'
 # TODO -e option
 
+### git blame
+complete -f -c git -n '__fish_git_needs_command' -a blame -d 'Show what revision and author last modified each line of a file'
+complete -f -c git -n '__fish_git_using_command blame' -s b -d 'Show blank SHA-1 for boundary commits'
+complete -f -c git -n '__fish_git_using_command blame' -l root -d 'Do not treat root commits as boundaries'
+complete -f -c git -n '__fish_git_using_command blame' -l show-stats -d 'Include additional statistics at the end of blame output'
+complete -f -c git -n '__fish_git_using_command blame' -s L -d 'Annotate only the given line range'
+complete -f -c git -n '__fish_git_using_command blame' -s l -d 'Show long rev'
+complete -f -c git -n '__fish_git_using_command blame' -s t -d 'Show raw timestamp'
+complete -f -c git -n '__fish_git_using_command blame' -s S -d 'Use revisions from revs-file instead of calling git-rev-list'
+complete -f -c git -n '__fish_git_using_command blame' -l reverse -d 'Walk history forward instead of backward'
+complete -f -c git -n '__fish_git_using_command blame' -s p -l porcelain -d 'Show in a format designed for machine consumption'
+complete -f -c git -n '__fish_git_using_command blame' -l line-porcelain -d 'Show the porcelain format'
+complete -f -c git -n '__fish_git_using_command blame' -l incremental -d 'Show the result incrementally'
+complete -f -c git -n '__fish_git_using_command blame' -l encoding -d 'Specifies the encoding'
+complete -f -c git -n '__fish_git_using_command blame' -l contents -d 'Annotates the changes starting backwards from the working tree copy'
```

Comment:
I think you've stopped reading too early. From the man page:

>This flag makes the command pretend as if the working tree copy has the contents of the named file

So something like "Instead of working tree, use the contents of the named file"?